### PR TITLE
Infinispan and Quarkus blog post

### PR DESCRIPTION
**Pull request by karesti**
A blog post highlighting the changes in Infinispan and Quarkus extension

### CODE REVIEW

**Comment by karesti** [comment]
Hey @gsmet @geoand @cescoffier since  3.11 is out, may we publish this blog post ? I'll update with any change you require on it

**Comment by geoand** [comment]
Makes sense to me!

File: _posts/2024-05-06-quarkus-infinispan-new.adoc [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: 'Quarkus and Infinispan: winner combo'
+date: 2024-04-29
```
Comment (gsmet) [inline _posts/2024-05-06-quarkus-infinispan-new.adoc:4]: This needs an update, together with the date in the file.

File: _posts/2024-05-06-quarkus-infinispan-new.adoc [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        String id = UUID.randomUUID().toString();
+        String description = String.format("Fruit in city %s, with name %s", city, name);
+        return new ExpensiveFruitsResponse(id, description, metadata);
+    }
+----
+
+Read all about it in the https://quarkus.io/guides/cache-infinispan-reference[Infinispan Cache Extension Guide]
+
+== Infinispan and Quarkus Langchain4j Integration
+Now, you can use Infinispan as an Embedding Store in the Quarkus Langchain 4J extension.
```
Comment (gsmet) [inline _posts/2024-05-06-quarkus-infinispan-new.adoc:124]:
```suggestion
Now, you can use Infinispan as an Embedding Store in the Quarkus LangChain4j extension.
```

File: _posts/2024-05-06-quarkus-infinispan-new.adoc [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+Infinispan has full text features thanks to a special integration with Hibernate Search.
+Plus, starting from Hibernate 7.1, Vector Search is supported. This means you can use the
```
Comment (gsmet) [inline _posts/2024-05-06-quarkus-infinispan-new.adoc:126]:
```suggestion
Plus, starting from Hibernate Search 7.1, Vector Search is supported. This means you can use the
```

**Review by gsmet — COMMENTED** [review]
I spotted a few minor things but otherwise it looks good to go.

Ping me loudly when fixed :).

**Comment by karesti** [comment]
@gsmet thanks for the review! I've changed the date

**Review by gsmet — APPROVED** [review]
I fixed the file name, let's merge!

**Comment by karesti** [comment]
Thanks @gsmet !!

**Pull request merged**